src/main/java/uk/gov/hmcts/reform/sscs/service/coversheet/CoversheetService.java

```diff
@@ -47,15 +48,20 @@ public Optional<byte[]> createCoverSheet(String identifier) {
                     String derivedTemplate = documentConfiguration.getEvidence()
                             .get(sscsCaseData.getLanguagePreference()).get(TEMPLATE);
                     Address address = sscsCaseData.getAppeal().getAppellant().getAddress();
+                    var lines = LetterUtils.lines(address);
+                    for (int i = lines.size(); i < 5; i++) {
+                        lines.add("");
```

Comment:
not sure I understand what this line is doing

Reply:
padding it to ensure there are 5 lines even if they are blank